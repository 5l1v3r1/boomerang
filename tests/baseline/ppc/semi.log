ERROR: attempted to decode outside any known segment 0x0x100104e0
ERROR: attempted to decode outside any known segment 0x0x10010444
after removing matching assigns (%pc).
in proc main adding addrExp r1{0} - 32 to local table
in proc main adding addrExp r1{0} - 4 to local table
in proc main adding addrExp r1{0} + 4 to local table
in proc main adding addrExp r1{0} - 24 to local table
in proc main adding addrExp r1{0} - 20 to local table
in proc main adding addrExp r1{0} - 20 to local table
in proc main adding addrExp r1{0} - 20 to local table
in proc main adding addrExp r1{0} - 20 to local table
in proc main adding addrExp r1{0} - 20 to local table
in proc main adding addrExp r1{0} - 20 to local table
in proc main adding addrExp r1{0} - 20 to local table
in proc main adding addrExp r1{0} - 20 to local table
in proc main adding addrExp r1{0} - 20 to local table
in proc main adding addrExp r1{0} - 20 to local table
in proc main adding addrExp r1{0} - 20 to local table
in proc main adding addrExp r1{0} - 20 to local table
in proc main adding addrExp r1{0} - 20 to local table
in proc main adding addrExp r1{0} - 20 to local table
in proc main adding addrExp r1{0} - 20 to local table
in proc main adding addrExp r1{0} - 20 to local table
in proc main adding addrExp r1{0} - 20 to local table
in proc main adding addrExp r1{0} - 20 to local table
### WARNING: iteration limit exceeded for dfaTypeAnalysis of procedure main ###
in proc main adding addrExp r1{0} - 32 to local table
in proc main adding addrExp r1{0} - 4 to local table
in proc main adding addrExp r1{0} + 4 to local table
in proc main adding addrExp r1{0} - 24 to local table
in proc main adding addrExp r1{0} - 20 to local table
in proc main adding addrExp r1{0} - 20 to local table
in proc main adding addrExp r1{0} - 20 to local table
in proc main adding addrExp r1{0} - 20 to local table
in proc main adding addrExp r1{0} - 20 to local table
in proc main adding addrExp r1{0} - 20 to local table
in proc main adding addrExp r1{0} - 20 to local table
in proc main adding addrExp r1{0} - 20 to local table
in proc main adding addrExp r1{0} - 20 to local table
in proc main adding addrExp r1{0} - 20 to local table
in proc main adding addrExp r1{0} - 20 to local table
in proc main adding addrExp r1{0} - 20 to local table
in proc main adding addrExp r1{0} - 20 to local table
in proc main adding addrExp r1{0} - 20 to local table
in proc main adding addrExp r1{0} - 20 to local table
in proc main adding addrExp r1{0} - 20 to local table
in proc main adding addrExp r1{0} - 20 to local table
in proc main adding addrExp r1{0} - 20 to local table
