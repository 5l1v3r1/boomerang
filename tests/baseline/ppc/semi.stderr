warning: textDelta different for section .got (ignoring).
decompiling entry point main
rtWARNING: no out edge for this BB in main:
L9: Invalid BB:
in edges: 10000508 
out edges: 
100104e0

WARNING: no out edge for this BB in main:
L10: Invalid BB:
in edges: 100004d8 
out edges: 
10010444

